File: app/src/main/java/com/usc/myway/Collection.java
// collections/album folder (item)
package com.usc.myway;

import java.util.ArrayList;
import java.util.List;

public class Collection {
    public String name;
    public String icon;
    public List<String> locationKeys; // list of "lat,lng" keys

    public Collection(String name, String icon) {
        this.name = name;
        this.icon = icon;
        this.locationKeys = new ArrayList<>();
    }
}

File: app/src/main/java/com/usc/myway/Collection.kt
// collections/album folder (item). `id` is the Firestore doc id under users/{uid}/collections.
package com.usc.myway

import java.util.UUID

class Collection(var name: String, var icon: String, val id: String = UUID.randomUUID().toString()) {
    val locationKeys: MutableList<String> = mutableListOf() // list of "lat,lng" keys
}
